src/mantine-demos/src/demos/core/Text/Text.demo.inherit.tsx

```diff
@@ -5,7 +5,13 @@ const code = `
 import { Text, Title } from '@mantine/core';
 
 function Demo() {
-  return <Title order={3}>Highlight <Text color="blue" inherit component="span">something</Text>in title</Title>;
+  return <Title order={3}>
```

Comment:
It is formatted incorrectly, should be `<Title order={3}>Title in which you want to<Text color="blue" inherit component="span">highlight</Text>something</Title>`

Reply:
let me update it